style.css

```diff
@@ -1,139 +1,642 @@
+  justify-content: center;
+}
+
+.about__title {
+  width: 5ch;
+}
+
+.about__dsc {
+  margin: 10px 0;
+}
+
+.about__connect {
+  text-transform: uppercase;
+}
+
+.about__icons__ul {
+  margin: 1.3rem 0 1rem;
+  align-items: center;
+  justify-content: space-between;
+  width: 11rem;
+}
+
+.about .btn {
+  width: 162px;
+  height: 48px;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  cursor: pointer;
+}
+
+.tools {
+  margin: 6px 0;
+  padding: 24px 0;
+}
+
+.langs__top {
+  align-items: center;
+  justify-content: space-between;
+  padding: 23px 40px 23px 0;
+  cursor: pointer;
+}
+
+.langs__top__img {
+  transform: rotate(90deg);
+}
+
+.langs__bottom li {
+  align-items: center;
+  gap: 1rem;
+  border-radius: 8px;
+  margin: 12px 0;
+  padding: 9px 12px;
+}
+
+.about__frameworks {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.about__frameworks__top {
+  align-items: center;
+  justify-content: space-between;
+  padding: 23px 40px 23px 0;
+  cursor: pointer;
+}
+
+.about__skills {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.about__skills__top {
+  align-items: center;
+  justify-content: space-between;
+  padding: 23px 40px 23px 0;
+  cursor: pointer;
+}
+
+.error {
+  text-align: center;
+  color: red;
+}
+
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
+
+.modal__header {
+  justify-content: space-between;
+}
+
+.modal__header img {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.modal__image {
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.modal__dsc {
+  flex-direction: column;
+}
+
+.modal__dsc__first {
+  margin: 0.7rem 0;
+}
+
+.modal__dsc__second {
   flex-direction: column;
+  align-items: flex-start;
+}
+
+.modal__dsc__second div {
+  margin: 1rem 0;
+}
+
+.modal__dsc__second p {
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin-right: 0.5rem;
+}
+
+.modal__btn {
+  margin: 1rem 0;
+}
+
+.modal__btn li a {
+  border: 1px solid #6070ff;
+  padding: 0.6rem 1.5vw;
+  border-radius: 8px;
+  margin-right: 0.5rem;
 }
 
-#contact ul {
+.modal__feature ul {
+  padding: 0;
+  margin: 0;
+}
+
+.modal__feature ul li {
+  margin-left: 1.5rem;
+}
+
+.modal.active {
+  visibility: visible;
+  transform: scale(1);
+  opacity: 1;
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  overflow: auto;
+  visibility: hidden;
   display: flex;
   align-items: center;
-  padding-left: 0;
-  margin-left: 24px;
+  justify-content: center;
+  opacity: 0;
+  transition: opacity 1s, transform 1s, visibility 0s;
+  transform: scale(0);
+  background: rgba(193, 199, 208, 0.5);
+  z-index: 200;
+}
+
+.overlay.active {
+  visibility: visible;
+  transform: scale(1);
+  opacity: 1;
+  pointer-events: all;
+}
+
+.navigation_bar__menu__link {
+  transition: color 0.5s ease-in-out, font-size 0.5s ease-in-out;
+}
+
+@media (min-width: 992px) {
```

Comment:
- [Optional] A milestone for desktop design is coming up. I will suggest you work on your project a milestone at a time.